SumoLogic.Logging.Log4Net/BufferedSumoLogicAppender.cs

```diff
@@ -181,6 +181,8 @@ public bool UseConsoleLog
             set; 
         }
 
+        protected override bool RequiresLayout => true;

```

Comment:
I'm not against use of newer syntax, but currently this is an isolated major style departure from all the rest of the project. I would request that you maintain the existing code style.